lib/unfurls/index.js

```diff
@@ -53,10 +54,19 @@ async function linkShared(req, res) {
         req.log.debug(err, 'Could not get unfurl attachment');
         return Promise.resolve();
       }
+      if (err.name === 'GitHubAccountNotConnected') {
+        await workspace.client.chat.postEphemeral({
+          channel: event.channel,
+          user: event.user,
+          attachments: [new PromptSignIn().getAttachment()],
+        });
+        return Promise.resolve();
+      }
       throw err;
     }
     if (hasEarlyAccess({ channelId: event.channel, teamId: req.body.team_id })) {
       if (newUnfurl && !newUnfurl.isDelivered) {
+        // todo: If app is not invited to channel, then send as DM to user
```

Comment:
What if the app didn't attempt to unfurl if it's not invited to the channel?

Either way, would it make sense to add this as a checkbox on #52 instead of an inline TODO?

Reply:
> What if the app didn't attempt to unfurl if it's not invited to the channel?

We don't have a way of knowing whether the app is in the channel until we try to send an ephemeral message, from what I can tell :/ 
Unfurling works fine even if the app is not in the channel

> Either way, would it make sense to add this as a checkbox on #52 instead of an inline TODO?

Yes, good point 👍 I'll add it as a comment #528